src/app/create/codebases/services/github.service.ts

```diff
@@ -67,9 +65,7 @@ export class GitHubService {
     if (this.cache.has(url)) {
       return this.cache.get(url);
     } else {
-      let res = this.getHeaders(this.getUserName(fullName))
```

Comment:
Can you please suggest how to fix the previous code to ensure the authentication header is set?